simple_history/locale/ur/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,134 @@
+# Urdu translation for django-simple-history
+# Copyright (C) 2023
+# This file is distributed under the same license as the django-simple-history package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: django-simple-history 3.3.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-07-02 11:10+0800\n"
+"PO-Revision-Date: 2023-07-02 11:50+0800\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: ur\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#: .\simple_history\admin.py:102
+#, python-format
+msgid "View history: %s"
+msgstr "تاریخ دیکھیں: %s"
+
+#: .\simple_history\admin.py:104
+#, python-format
+msgid "Change history: %s"
+msgstr "تاریخ کو تبدیل کریں: %s"
+
+#: .\simple_history\admin.py:110
+#, python-format
+msgid "The %(name)s \"%(obj)s\" was changed successfully."
+msgstr "%(name)s \"%(obj)s\" کامیابی سے تبدیل کر دیا گیا."
+
+#: .\simple_history\admin.py:116
+msgid "You may edit it again below"
+msgstr "آپ نیچے اس میں دوبارہ ترمیم کر سکتے ہیں"
+
+#: .\simple_history\admin.py:217
+#, python-format
+msgid "View %s"
+msgstr "دیکھیں %s"
+
+#: .\simple_history\admin.py:219
+#, python-format
+msgid "Revert %s"
+msgstr "تبدیلی واپس کریں %s"
+
+#: .\simple_history\models.py:552
+msgid "Created"
+msgstr "بنایا"
+
+#: .\simple_history\models.py:552
+msgid "Changed"
+msgstr "بدل گیا"
+
+#: .\simple_history\models.py:552
+msgid "Deleted"
+msgstr "حذف کر دیا گیا"
+
+#: .\simple_history\templates\simple_history\_object_history_list.html:9
+msgid "Object"
+msgstr "آبجیکٹ"
+
+#: .\simple_history\templates\simple_history\_object_history_list.html:13
+msgid "Date/time"
+msgstr "تاریخ/وقت"
+
+#: .\simple_history\templates\simple_history\_object_history_list.html:14
+msgid "Comment"
+msgstr "تبصرہ"
+
+#: .\simple_history\templates\simple_history\_object_history_list.html:15
+msgid "Changed by"
+msgstr "کی طرف سے تبدیل"
+
+#: .\simple_history\templates\simple_history\_object_history_list.html:16
+msgid "Change reason"
+msgstr "تبدیلی کا سبب"
+
+#: .\simple_history\templates\simple_history\_object_history_list.html:37
+msgid "None"
+msgstr "کوئی نہیں"
+
+#: .\simple_history\templates\simple_history\object_history.html:11
+msgid ""
+"Choose a date from the list below to revert to a previous version of this "
+"object."
+msgstr ""
+"اس آبجیکٹ کے پچھلے ورژن پر واپس جانے کے لیے نیچے دی گئی فہرست میں سے ایک تاریخ کا انتخاب کریں۔"
```

Comment:
Not sure how punctuation is used in Urdu, but it looks like there's a missing period (`.`) at the end of this sentence (considering that one was included on line 33) 🤔 Is this correct? If so, could you also add the missing periods on line 94, 118 and 122 below as well? 🙂

Reply:
Sure, I will make periods consistent

Reply:
Done

Reply:
Oh wait, I just noticed from the diff in 332f61050f5c0ab98480c7ecebddb07ce25d45fb that you removed `۔` from the beginning of the mentioned sentences... So I'm assuming the original sentences already had proper punctuation..? In that case, it would be better to revert 332f61050f5c0ab98480c7ecebddb07ce25d45fb 😅 (and also do the same change for line 33, for consistency - **but only if** that would be more correct in Urdu.)

Reply:
So, Urdu is read from right to left. I added period at the right of sentence #33 as it will not be a complete Urdu sentence because of variables.  Otherwise, in Urdu periods should come at the left of sentence.

Reply:
I see, so reverting 332f61050f5c0ab98480c7ecebddb07ce25d45fb would make the most proper punctuation?

Reply:
But, I have taken leap from[ arabic translation](https://github.com/jazzband/django-simple-history/blob/master/simple_history/locale/ar/LC_MESSAGES/django.po) (it is also read from right to left), and periods are in the right of sentence.

Reply:
Hm okay, so do you think it's good enough as it is now? If so, I'll merge 🙂

Reply:
Yes. Thank you for reviewing